exercises/introducao-a-javascript-e-logica-de-programacao_1/exe01.js

```diff
@@ -1,8 +1,22 @@
-let a = 10;
-let b = 25;
 
-console.log('Soma: ' + (a + b));
-console.log('Subtração: ' + (a - b));
-console.log('Multiplicação: ' + (a * b));
-console.log('Divisão: ' + (a / b));
+function sum(valueOne, ValueTwo) {
+  let result = valueOne + ValueTwo;
+  return result;
```

Comment:
```suggestion
  return result = valueOne + ValueTwo;
```
Você não precisa criar uma váriavel nesse caso, pode apenas retornar o valor + valor2.
Para as outras funções também, creio que compensa criar a variável caso for ter que fazer algo mais complexo. Mas está muito bom.

Reply:
Obrigado pelas correções, não tinha pensado dessa forma, fica bem mais performático na forma que você mostrou. Já que não precisamos criar váriaveis desnecessárias.